apps/meteor/client/components/Omnichannel/Tags.tsx

```diff
@@ -23,13 +22,18 @@ const Tags = ({
 	const t = useTranslation();
 	const forms = useFormsSubscription() as any;
 
-	const { value: tagsResult, phase: stateTags } = useEndpointData('/v1/livechat/tags');
-
 	// TODO: Refactor the formsSubscription to use components instead of hooks (since the only thing the hook does is return a component)
 	const { useCurrentChatTags } = forms;
 	// Conditional hook was required since the whole formSubscription uses hooks in an incorrect manner
 	const EETagsComponent = useCurrentChatTags?.();
 
+	const getTags = useEndpoint('GET', '/v1/livechat/tags');
+	const {
+		data: tagsResult,
+		isLoading,
+		isFetching,
```

Comment:
You can use here `isInitialLoading` which is equivalent to `isFetching && isLoading`